lib/ansible/module_utils/basic.py

```diff
@@ -1455,7 +1455,12 @@ def _check_arguments(self, spec=None, param=None, legal_inputs=None):
             msg = "Unsupported parameters for (%s) module: %s" % (self._name, ', '.join(sorted(list(unsupported_parameters))))
             if self._options_context:
                 msg += " found in %s." % " -> ".join(self._options_context)
-            msg += " Supported parameters include: %s" % (', '.join(sorted(spec.keys())))
+            supported_parameters = list()
+            for key in spec.keys():
+                supported_parameters.append(key)
+                if 'aliases' in spec[key] and spec[key]['aliases']:
+                    supported_parameters.extend(spec[key]['aliases'])
+            msg += " Supported parameters include: %s" % (', '.join(sorted(supported_parameters)))
```

Comment:
i would add the aliases to the options in this way:

option1 (alias1, alias2), option2 (alias1, ....

Or simpler to direct them to `ansible-doc -l <module name>` for 'valid options'

Reply:
Yes, I thought of the `option1 (alias1)...` idea. I amend the change.